src/nunit.analyzers.tests/ClassicModelAssertUsage/IsNotNullAndNotNullClassicModelAssertUsageCodeFixTests.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Immutable;
+using Gu.Roslyn.Asserts;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Analyzers.ClassicModelAssertUsage;
+using NUnit.Analyzers.Constants;
+using NUnit.Framework;
+
+namespace NUnit.Analyzers.Tests.ClassicModelAssertUsage
+{
+    [TestFixture]
+    public sealed class IsNotNullAndNotNullClassicModelAssertUsageCodeFixTests
+    {
+        private static readonly DiagnosticAnalyzer analyzer = new ClassicModelAssertUsageAnalyzer();
+        private static readonly CodeFixProvider fix = new IsNotNullAndNotNullClassicModelAssertUsageCodeFix();
+
+        [Test]
+        public void VerifyGetFixableDiagnosticIds()
+        {
+            var fix = new IsNotNullAndNotNullClassicModelAssertUsageCodeFix();
+            var ids = fix.FixableDiagnosticIds.ToImmutableArray();
+
+            Assert.That(ids.Length, Is.EqualTo(2), nameof(ids.Length));
+            Assert.That(ids, Contains.Item(AnalyzerIdentifiers.IsNotNullUsage), nameof(AnalyzerIdentifiers.IsNotNullUsage));
+            Assert.That(ids, Contains.Item(AnalyzerIdentifiers.NotNullUsage), nameof(AnalyzerIdentifiers.NotNullUsage));
+        }
+
+        [TestCase("IsNotNull", AnalyzerIdentifiers.IsNotNullUsage)]
+        [TestCase("NotNull", AnalyzerIdentifiers.NotNullUsage)]
+        public void VerifyIsNotNullAndNotNullFixes(string assertion, string diagnosticId)
+        {
+            var expectedDiagnostic = ExpectedDiagnostic.Create(diagnosticId);
+
+            var code = TestUtility.WrapMethodInClassNamespaceAndAddUsings($@"
```

Comment:
you can use `TestUtility.WrapInTestMethod` to make it a bit clearer

Reply:
I used `IsTrueAndTrueClassicModelAssertUsageCodeFixTests.cs` as a template. Should the change be made in both places?

Reply:
Same comment as above :). I agree with @Dreamescaper, but I also think it is better to solve all the occurrences in one go (and not part of this addition).